setup.sh

```diff
@@ -17,3 +17,7 @@ else
     chmod +x updater.sh
     printf "${GREEN}\nAlias 'up' added.\nUse 'up' or run './updater.sh'.\n\n${GREEN}Setup completed.\n${NORMAL}"
 fi
+
+read -p "${RED}Press enter, the terminal will be closed: open a new terminal to see changes.${NORMAL}" text
```

Comment:
I've seen that in my case the terminal won't close but the concept is still that. I'll modify this string